roles/ceph-common/templates/restart_osd_daemon.sh.j2

```diff
@@ -13,7 +13,11 @@ check_pgs() {
     let RETRIES=RETRIES-1
   done
   # PGs not clean, exiting with return code 1
-  echo "Error with PGs, check config"
+  echo "Error while running `ceph $CEPH_CLI -s`, PGs were not reported as active+clean"
```

Comment:
use single quotes instead of backquote, otherwise the code command will be executed inside the `echo`.